app/assets/javascripts/discourse/app/models/topic.js

```diff
@@ -376,17 +376,31 @@ const Topic = RestModel.extend({
     return ajax(`/t/${this.id}/remove_bookmarks`, { type: "PUT" });
   },
 
+  bookmarkCount: alias("bookmarks.length"),
```

Comment:
Before when calculating bookmarks count we were checking everywhere if `bookmarks` is not null, probably we still need this check in this property. If we're sure we're always going to have the bookmarks array, you're version is better, but if not:
```suggestion
  @discourseComputed
  bookmarkCount(bookmarks) {
    return  bookmarks ? bookmarks.length : 0;
  },
```

Reply:
We should always have at least an empty array because of this in topic_view, so this should be fine:

https://github.com/discourse/discourse/blob/d249d5355bf6c7f01baad8b60eb02bfbb704696b/lib/topic_view.rb#L398-L403